Automattic/wp-calypso
# A4A: Sites Dashboard - Preview Pane - Hosting Feature design

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/263

## Proposed Changes

* Update the design of the Hosting feature in the preview pane. The design was updated to match the provided Figma file:  1UpoqS1KkCtLXVX68TbMSM-fi
* Change the php_version_num field to a string, since it contains semantic versioning information
* Add new fields to be displayed in the Hosting feature: `wordpress_version` and `hosting_provider_guess`

**Note**: This PR does not update the 'Status' information. Using the `public` field proved to be unreliable. This information will have to be updated in a subsequent PR. Context: https://github.com/Automattic/automattic-for-agencies-dev/issues/264 

## Testing Instructions

* Run `yarn start-a8c-for-agencies`
* Go to `http://agencies.localhost:3000/sites`
* Click over a site to open the preview pane
* The "Hosting" tab should show the design matching the Figma file
* The information should be correct, with the exception of `Status`
* Check if the logo is shown for WordPress.com and Pressable sites

![image](https://github.com/Automattic/wp-calypso/assets/37049295/f4d809e4-cd09-4ebe-a254-bbb517a36439)

## Pre-merge Checklist
